### DISCUSSION TITLE
macOS Ventura

### DISCUSSION THREAD

**Discussion by SkyrilHD** [SkyrilHD/Dell-Exx50-Hackintosh#57] [General]
With the new release of EFI (v3.x) macOS Ventura is now officially supported and everything related to it can be chatted here.

**Comment by ghost** [comment]
EFI works perfectly when creating the update to Ventura USB installer perfectly. People MUST follow your directions to the T. In my experience, I have had good and bad installs via OTA. It is better to use an installer (either via downloading the gibMacOS script) or via creating an installer with that .app image on a bootable usb. Under no circumstances have been able to get the update to work through "System Preferences". The methods above work on the E7450. The GPU patch via opencore-patcher works as well.
